packages/pelagos/src/components/Select.less

```diff
@@ -43,6 +43,7 @@
 
 		&[aria-disabled='true'] {
 			color: var(--text-disabled);
+			background-color: var(--field);
```

Comment:
This was needed to prevent the `&:hover` background colour from being used when the select is disabled